tests/testthat/test-plot.tbl_uvregression.R

```diff
@@ -0,0 +1,18 @@
+skip_if_not(broom.helpers::.assert_package("ggstats", pkg_search = "gtsummary", boolean = TRUE))
+
+test_that("plot.tbl_regression() works", {
+  expect_silent(
+    tbl1 <- tbl_uvregression(
+      trial,
+      x = trt,
+      include = c(marker, age),
+      show_single_row = trt,
+      method = lm
+    )
+  )
+  expect_error(
```

Comment:
Im never really sure how to test plotting objects in r packages, is there some sort of slew of tests you want me to more rigorously check